built-in-references/KubernetesService/container-allowed-images/limited-preview/azurepolicy.json

```diff
@@ -0,0 +1,47 @@
+{
+  "properties": {
+    "displayName": "[Limited Preview]: Ensure only allowed container images in AKS",
+    "policyType": "BuiltIn",
+    "mode": "Microsoft.ContainerService.Data",
+    "description": "This policy ensures only allowed container images are running in an Azure Kubernetes Service cluster. Limited Preview policies only work for registered subscriptions. To register, please go to https://aka.ms/akspolicyonboarding. For instruction on using this policy, please go to https://aka.ms/akspolicydoc.",
```

Comment:
>https://aka.ms/akspolicydoc [](start = 300, length = 27)

when will this turn into a useful doc?

Reply:
David is publishing a new doc soon.

Reply:
Built-in definitions and the link are live now in Prod!